conf.yaml

```diff
@@ -685,3 +685,9 @@ targets:
       - app.netlify.com
       - netlify.app
     twitter: "@Netlify"
+  Flashback:
+    href: https://www.flashback.org/
+    hosts:
+      - www.flashback.org
+      - static.flashback.org
+    twitter: "@"  
```

Comment:
If they don't have a Twitter account, please completly remove the `twitter` key from the entry.

Reply:
Still not fixed